modules/apps/frontend-js/frontend-js-test/src/testFunctional/paths/A11yTool.path

```diff
@@ -89,6 +89,37 @@
 	<td>//*[contains(@class, 'clay-popover-bottom show')]//*[contains(@class,'text-secondary') and contains(.,'${key_impact}')]</td>
 	<td></td>
 </tr>
+<!--SIDE PANEL COMPONENT VIEW-->
+<tr>
+	<td>VIOLATION_PANEL_COMPONENT_TITLE</td>
+	<td>//*[contains(@class,'sidebar-header')]//*[contains(@class,'component-title') and contains(.,'${key_text}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>VIOLATION_PANEL_COMPONENT_IMPACT</td>
+	<td>//*[contains(@class,'sidebar-header')]//*[contains(@class,'component-subtitle') and contains(.,'${key_impact}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>VIOLATION_PANEL_COMPONENT_LINK</td>
+	<td>//*[contains(@class,'sidebar-header')]//*[contains(@class,'component-subtitle') and contains(.,'More Info')]//*[contains(@href,'')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>VIOLATION_PANEL_COMPONENT_HEADER</td>
+	<td>//*[contains(@class,'panel-header')]//*[contains(@class,'panel-title') and contains(.,'${key_header}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>VIOLATION_PANEL_COMPONENT_BODY</td>
+	<td>//*[contains(@class,'panel-body') and contains(.,'${key_text}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>VIOLATION_PANEL_OCCURRENCE_LIST_ITEM</td>
+	<td>//*[contains(@class,'panel-body')]//ul//*[contains(@class,'list-group-item') and contains(.,'${key_text}')]</td>
+	<td></td>
+</tr>
```

Comment:
What about including or replacing the first part of xpath with `//*[contains(@class,'a11y-panel')]`? I think the other classes could be too generic and target other side panel elements on a given page.

![image](https://user-images.githubusercontent.com/35235266/142069976-7412387d-9d6e-4e7c-b714-aaa13004d99e.png)

Reply:
yeah I can add that :+1:

Reply:
Cool, I think this is ok to forward after this change. :)